frontend/src/main/scala/bloop/testing/TestSuiteEvent.scala

```diff
@@ -6,9 +6,11 @@ import ch.epfl.scala.bsp
 import ch.epfl.scala.bsp.BuildTargetIdentifier
 import ch.epfl.scala.bsp.endpoints.{Build, BuildTarget}
 import sbt.testing.{Event, Status}
+import cats.syntax.option._
```

Comment:
Let's avoid the use of this import and instead do a manual conversion ourselves, the use case here is to overkill to have a wildcard import on cats :wink: